.github/workflows/multichain-contract.yml

```diff
@@ -1,42 +0,0 @@
-name: Multichain Contract
```

Comment:
How do we deploy the contract now?

Reply:
By using [deploy-multichain-dev-contract.yml](https://github.com/near/mpc/pull/872/files#diff-c836f17af97ffd34fd2582c700d283714a220669cd924785e34acc15255d2fe3), I don't think this workflow file is in use.